### DISCUSSION TITLE
Odpytanie o daną kategorię powinno zawierać pole o wymagalności produktyzacji/progu wymagalności

### DISCUSSION THREAD

**Discussion by hahyes** [allegro/allegro-api#5937] [Suggestions - rejected]
Witam,

dopracowuję system produktyzacji po swojej stronie i mały dylemat - skąd mam wiedzieć, jaka kategoria potrzebuje koniecznie produktów? Wiem, że jest arkusz z wymienionymi kategoriami, w których od 1 sierpnia mają być obowiązkowo produkty (tj. 100%) ale przyznajcie, że implementowanie do systemu obsługi tego arkusza, żeby zaimportować z niego dane to... no przesada lekko mówiąc. Biorąc pod uwagę, że nie wszystkie kategorie na ten moment zostaną przełączone w tryb 100% od 1 sierpnia sugeruję, aby istniało odzwierciedlenie tego w API, gdy odpytujemy `/sale/categories/{id}`. Pole `options` to chyba byłoby odpowiednie miejsce na dodanie takiego parametru, mamy tutaj `productCreationEnabled` choćby. 

Chyba, że `productCreationEnabled` decyduje o wymagalności? Ale to raczej nie jest prawda, ponieważ o ile dobrze pamiętam sugerując nowe produkty, kategoria bez `productCreationEnabled` spowoduje błąd.

**Comment by Lukasz-Zurek** [comment]
Pole:`productCreationEnabled` nie decyduje o wymagalności. Tym bardziej opisana przez Ciebie propozycja ma rację bytu. Dzięki za sugestię.

**Comment by MartaNowaczyk** [comment]
W 2024 roku chcemy by allegro stało się serwisem opartym na produktach (100% produktyzacji), stąd nie planujemy wprowadzać  zmian w tym zakresie.

**Reply by hahyes** [reply]
Fair.
